actors/rozetka-count/Dockerfile

```diff
@@ -0,0 +1,36 @@
+# Dockerfile contains instructions how to build a Docker image that will contain
+# all the code and configuration needed to run your actor. For a full
+# Dockerfile reference, see https://docs.docker.com/engine/reference/builder/
+
+# First, specify the base Docker image. Apify provides the following base images
+# for your convenience:
+#  apify/actor-node-basic (Node.js on Alpine Linux, small and fast image)
+#  apify/actor-node-chrome (Node.js + Chrome on Debian)
+#  apify/actor-node-chrome-xvfb (Node.js + Chrome + Xvfb on Debian)
+# For more information, see https://apify.com/docs/actor#base-images
+# Note that you can use any other image from Docker Hub.
+FROM apify/actor-node-chrome:v0.21.10
```

Comment:
What is the version 0.21.10 doing here (that is quite old right now)? Let's use the new Dockerfiles actor-node-puppeteer-chrome

Reply:
Actually, since you use cheerio, do we need chrome anywhere? If not, let's use `actor-node`

Reply:
Yeah, this is weird to me too! I was forced to specify a certain tag for the Docker image, because the automatic code checker here didn't allow me to leave it neither without a tag (e.g. simply **FROM apify/actor-node-chrome**) nor use the "latest" tag. So I specified the only tag I could find on the Docker image web page. What would be better in this case?

Reply:
@rarous ?

Reply:
Please, see other actor in this repo for reference

Reply:
OK, so I'll just change the image to **apify/actor-node:14** as it is in other ones. Thanks!